### ISSUE TITLE
Asus Zenbook UX303LN not all function keys work.

### ISSUE THREAD

**Issue by okristian1** [agajews/ApricityOS#39]
Fn + F5 or F6 adjust backlight in windows. Non responsive in Apricity OS. Backlight is adjustable trough terminal commands or with brightness slider in gnome.

Fn + F9 Should toggle the touchpad ON/OFF. OSD graphic indicating buttonpress is shown but touchpad state is not toggled.

**Comment by habutre** [comment]
All of @okristian1  said happens to me and more, the zenbook **home,end,pgdn and pdup** keys whose are located on arrow keys and activated by `Fn` don't work too.
I've no tested yet, but follow the instructions [here](https://wiki.archlinux.org/index.php/ASUS_Zenbook_UX303#Function_Keys)  might put the backlight to work

**Comment by agajews** [comment]
I was just about to link the archwiki page for you when I noticed you put it in your post. The instructions on `Function Keys` will probably fix your issue.

**Comment by habutre** [comment]
Yep, fix lots but not all. I'll still try to fix the remaining ones. I figure out the arrow keys + Fn just don't work on terminal, on chrome or nautilus they're working fine.
The airplane mode is working to turn ON, but failing to turn OFF, a great thing is the menu shows the airplane mode enabled allowing to turn it OFF (again thx ApricityOS, I'm glad to find you)

Now, I still confirm the @okristian1 claims about touchpad (un)lock and backlight.
